src/processors/JsonSchemaInputProcessor.ts

```diff
@@ -47,13 +49,21 @@ export class JsonSchemaInputProcessor extends AbstractInputProcessor {
 
     /**
      * Process a draft 7 schema
+     * 
      * @param input to process as draft 7
      */
-    private processDraft7(input: any) : CommonInputModel {
-        const schema = Schema.toSchema(input);
+    private async processDraft7(input: any) : Promise<CommonInputModel> {
+        const refParser = new $RefParser;
```

Comment:
Is new parser needed for every parsing?

Reply:
Ehhh, goood question. I guess not? 😄

Reply:
let me try

Reply:
Ahh, it should be new 😄 I remember one issue in parser-js that some state in refParser was reassigned in asynchronus functions, so we should stay with it.